src/modules/@git/fetch.ts

```diff
@@ -0,0 +1,64 @@
+import { createSpinner } from "nanospinner";
+import { sync as spawnSync } from "cross-spawn";
+import { Module } from "../../types";
+import chalk from "chalk";
+import { is } from "../../lib/utils";
+
+export default {
+    validate: (config, cwd) => is.set(config.repository),
+    async initiate(config, cwd) {
+        const gitSpinner = createSpinner("Checking git...");
+        gitSpinner.start();
+        if (await spawnSync("git", ["-v"], { cwd }).error) {
+            gitSpinner.error({ text: "Git is not installed" });
+        } else {
+            let remote = spawnSync("git", ["remote", "get-url", "origin"], {
+                cwd,
+            });
+            if (remote.status !== 0) {
+                gitSpinner.update({ text: "Initializing repository" });
+                await spawnSync("git", ["init"], { cwd });
+            }
+            remote = spawnSync("git", ["remote"], {
+                cwd,
+            });
+            if (remote.status !== 0)
+                return gitSpinner.error({
+                    text: "[!] Error: Couldn't initialize git Repository",
+                });
+            gitSpinner.update({ text: "Fetching repository" });
+            if (remote.stdout.toString() !== config.repository) {
+                if (remote.stdout.toString() === "") {
+                    if (
+                        spawnSync(
+                            "git",
+                            [
+                                "remote",
+                                "add",
+                                "origin",
+                                `"${config.repository}"`,
+                            ],
+                            { cwd }
+                        ).status !== 0
```

Comment:
This is the mistake.

Reply:
oh, it is

Reply:
You see, not my mistake 😁

Reply:
I added that, cuz i forgor that it isn't executed like in the shell

Reply:
Good that I reviewed it in detail 😅